app/models/series.rb

```diff
@@ -88,11 +88,8 @@ def Series.last_observation_buckets(frequency)
     {:last_observations => obs_buckets, :last_modifications => mod_buckets}
   end
   
-  def Series.all_names
-    all_names_array = []
-    all_names = Series.where(universe: 'UHERO').select(:name).all
-    all_names.each {|s| all_names_array.push(s.name)}
-    all_names_array
+  def Series.all_names(universe = 'UHERO')
+    Series.where(universe: universe).pluck(:name)
```

Comment:
ouch. I'm sure I wrote the original. Thanks for cleaning that up.

Reply:
No, it was Ben :=) Thanks for the approve.